README.rst

```diff
@@ -15,6 +15,14 @@ command and for specific subcommands through :code:`dcos <subcommand> --help`.
 Additional documentation for the CLI and for the DCOS in general is available
 in the `Mesosphere docs`_.
 
+To run the DCOS CLI as a container, you can pull it from the docker hub with::
+
+    docker run -i -t mesosphere/dcos-cli <url of your cluster>
```

Comment:
see my comment in slack.

Doing so would preclude us from running like this (providing a url and getting a bash prompt), which I think is a much more common use case (by far, for me at least).

You at least have to configure your container with the URL somehow.